sympy/integrals/tests/test_prde.py

```diff
@@ -270,13 +327,3 @@ def test_parametric_log_deriv():
     assert parametric_log_deriv_heu(Poly(5*t**2 + t - 6, t), Poly(2*x*t**2, t),
     Poly(-1, t), Poly(x*t**2, t), DE) == \
         (2, 6, t*x**5)
-
-
-def test_issue_10798():
```

Comment:
The test has **not** been removed, it has been shifted to [here](https://github.com/sympy/sympy/pull/12743/files#diff-af61f7566bec5996f0ce919794ab7413R152), i.e. in the test function `test_prde_cancel_liouvillian`.

The test is present is in right file in the sense that it is testing issue #10798 which was an issue with `is_deriv_k` though for that issue a more lower-level test could have been added i.e. we can replace the above test of `risch_integrate` with:
```
>>> fa, fd = Poly(-1, t0, domain='ZZ'), Poly(y, t0, domain='ZZ[y]')
>>> DE = DifferentialExtension(extension={'D': [Poly(1, y, domain='ZZ'), Poly(-1/y, t0, domain='ZZ(y)')], 'E_K': [], 'E_args': [], 'L_K': [1], 'L_args': [1/y]})
>>> is_deriv_k(fa, fd, DE)
([(t0, 1)], t0, -1)
```

But my point to keep the original test of `risch_integrate` is: as it tests the issue #10798 also it tests the `prde_cancel()`.